WordPress/twentytwentyfive
# Add pattern events - three columns with images and title (static)

**Description**

Fixes #84 https://github.com/WordPress/twentytwentyfive/issues/84
Adding the ‘events three columns with images and titles’ pattern.

At first I implemented this pattern as a dynamic one using a Grid Custom Query Loop block, which is in another branch of my forked repo. The problem is the custom query has to be configured targeting the desired pieces of content (events). I guess for a default theme is better as an static pattern.

**Screenshots**

https://github.com/user-attachments/assets/c79346c8-e86a-45eb-9138-ec5205770142


**Testing Instructions**

1. Create a page
2. Add the pattern
3. Confirm it looks like the design
